libobs/util/sse-intrin.h

```diff
@@ -25,8 +25,17 @@
 #ifndef WIN32_LEAN_AND_MEAN
 #define WIN32_LEAN_AND_MEAN
 #endif
+
+#if defined(_MSC_VER) && defined(__cplusplus)
+#include <cmath>
+#endif
+
+#if defined(__APPLE__) && !defined(__arm64__)
+#include <immintrin.h>
+#endif
```

Comment:
We didn't need to include this before for Intel Macs - did it fail to compile without this header? IIRC `immintrin` represents the `avx` class of vector functions.

Reply:
Answered by mail so the comment is out of the conversation.
https://github.com/obsproject/obs-studio/pull/10961#issuecomment-3145557980